app/models/ext_management_system.rb

```diff
@@ -422,6 +422,47 @@ def enable!
     update!(:enabled => true)
   end
 
+  def self.destroy_queue(ids)
+    ids = ids.to_miq_a
+    _log.info("Queuing destroy of #{name} with the following ids: #{ids.inspect}")
+    ids.each do |id|
```

Comment:
do we really have to send a different message for every destroy?
Better yet, could we just send in the where clause and not be `id` centric?